llvm/test/CodeGen/SPIRV/llvm-intrinsics/constrained-convert.ll

```diff
@@ -0,0 +1,181 @@
+    ret void
+}
+define dso_local spir_kernel void @test8(i16 %in) {
+entry:
+    %conv7 = tail call float @llvm.experimental.constrained.uitofp.f32.i16(i16 %in, metadata !"round.upward", metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test9(i16 %in) {
+entry:
+    %conv8 = tail call double @llvm.experimental.constrained.uitofp.f64.i16(i16 %in, metadata !"round.downward", metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test10(<4 x i32> %in) {
+entry:
+    %conv9 = tail call <4 x float> @llvm.experimental.constrained.uitofp.v4f32.v4i32(<4 x i32> %in, metadata !"round.tonearest", metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test11(float %in) {
+entry:
+    %conv10 = tail call i32 @llvm.experimental.constrained.fptosi.i32.f32(float %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test12(double %in) {
+entry:
+    %conv11 = tail call i32 @llvm.experimental.constrained.fptosi.i32.f64(double %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test13(float %in) {
+entry:
+    %conv12 = tail call i16 @llvm.experimental.constrained.fptosi.i16.f64(float %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test14(double %in) {
+entry:
+    %conv13 = tail call i16 @llvm.experimental.constrained.fptosi.i16.f64(double %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test15(<4 x double> %in) {
+entry:
+    %conv14 = tail call <4 x i16> @llvm.experimental.constrained.fptosi.v4i16.v4f64(<4 x double> %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test16(float %in) {
+entry:
+    %conv15 = tail call i32 @llvm.experimental.constrained.fptoui.i32.f32(float %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test17(double %in) {
+entry:
+    %conv16 = tail call i32 @llvm.experimental.constrained.fptoui.i32.f64(double %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test18(float %in) {
+entry:
+    %conv17 = tail call i16 @llvm.experimental.constrained.fptoui.i16.f32(float %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test19(double %in) {
+entry:
+    %conv18 = tail call i16 @llvm.experimental.constrained.fptoui.i16.f64(double %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test20( <4 x double> %in) {
+entry:
+    %conv19 = tail call <4 x i32> @llvm.experimental.constrained.fptoui.v4i32.v4f64(<4 x double> %in, metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test21(float %in) {
+entry:
+    %conv20 = tail call double @llvm.experimental.constrained.fpext.f64.f32(float %in,  metadata !"fpexcept.strict") 
+    ret void
+}
+define dso_local spir_kernel void @test22(<4 x float> %in) {
```

Comment:
Can you name the combination of types in the function names instead of just test<number>